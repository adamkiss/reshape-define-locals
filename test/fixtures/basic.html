<define-locals>
	bar: 'foo'
</define-locals>
<p class="{{ foo }}">{{ bar }}</p>