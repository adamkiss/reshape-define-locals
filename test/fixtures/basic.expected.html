<p class="bar">foo</p>
